src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -789,26 +763,24 @@ private void OnRectangleMouseEnter(object sender, EventArgs e)
 
```

Comment:
Below, no one should directly modify the value of `NodeModel` properties: `IsVisible`, `IsUpstreamVisible` and `ArgumentLacing`.